pandas/core/reshape.py

```diff
@@ -971,6 +971,11 @@ def get_dummies(data, prefix=None, prefix_sep='_', dummy_na=False,
         Otherwise returns a DataFrame with some SparseBlocks.
 
         .. versionadded:: 0.16.1
+    drop_first : bool, defalt False
+        Whether to get n-1 dummies out of n categorical levels by removing the
```

Comment:
in `reshaping.rst` you say `k-1`.  Maybe change one of them to match (both `k-1` or both `n-1`)?